packages/mobile/src/screens/app-screen/useAppScreenOptions.tsx

```diff
@@ -109,133 +114,129 @@ export const useAppScreenOptions = (
     navigation.navigate('trending')
   }, [navigation])
 
-  const handlePressSearch = useCallback(() => {
-    navigation.push('Search')
-  }, [navigation])
-
   const { isEnabled: isEarlyAccess } = useFeatureFlag(FeatureFlags.EARLY_ACCESS)
 
-  const screenOptions: (options: {
-    navigation: NativeStackNavigationProp<AppScreenParamList>
-    route: RouteProp<ParamListBase>
-  }) => NativeStackNavigationOptions = useCallback(
-    ({ navigation, route }) => {
-      // The manual typing is unfortunate here. There may be a better way, but
-      // the tricky bit is that StackNavigationOptions aren't known to the RouteProp.
-      // A better solution may be to wrap <Stack.Screen> in our own variant that
-      // can do some better generics & inference.
-      const params = route.params
-      // Notifications uses this in order to remove animations when going from the drawer
-      // to a nested stack screen.
-      const isFromNotifs =
-        params &&
-        'fromNotifications' in params &&
-        (params as ContextualParams).fromNotifications
-
-      return {
-        animation: isFromNotifs ? 'none' : 'default',
-        gestureEnabled: !isFromNotifs,
-        fullScreenGestureEnabled: true,
-        detachPreviousScreen: false,
-        cardOverlayEnabled: true,
-        cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS,
-        headerShadowVisible: false,
-        headerTitleAlign: 'center',
-        headerBackVisible: false,
-        headerLeft: (props) => {
-          const { canGoBack, ...other } = props
-          if (canGoBack) {
+  const screenOptions: (options: Options) => NativeStackNavigationOptions =
+    useCallback(
+      (options) => {
+        const { navigation, route } = options
+        // The manual typing is unfortunate here. There may be a better way, but
+        // the tricky bit is that StackNavigationOptions aren't known to the RouteProp.
+        // A better solution may be to wrap <Stack.Screen> in our own variant that
+        // can do some better generics & inference.
+        const params = route.params
+        // Notifications uses this in order to remove animations when going from the drawer
+        // to a nested stack screen.
+        const isFromNotifs =
+          params &&
+          'fromNotifications' in params &&
+          (params as ContextualParams).fromNotifications
+
+        const handlePressSearch = () => navigation.push('Search')
```

Comment:
this is the important line @raymondjacobson

Reply:
ty makes sense